hydradx/model/amm/arbitrage_agent.py

```diff
@@ -3,7 +3,7 @@
 
 
 # note that this function mutates state, test_agent, test_cex, and max_liquidity
-def process_next_swap(state, test_agent, test_cex, tkn_pair, ob_tkn_pair, buffer, max_liquidity, iters):
+def process_next_swap(state, test_agent, test_cex, tkn_pair, ob_tkn_pair, buffer, max_liquidity_dex, max_liquidity_cex, iters):
```

Comment:
Minor quibble, but using 'test_dex' as the variable name here would seem more consistent.

Reply:
Made this change. The "text_" names are an artifact from before a recent refactor but I will just leave them consistent for now.